### DISCUSSION TITLE
GLM-4-Flash 是  glm4-9b 吗？

### DISCUSSION THREAD

**Discussion by ponponon** [THUDM/GLM-4#603] [Q&A] [answered]
因为找不到 GLM-4-Flash 相关的地方询问，所以只能来 GLM-4 地方提问了

我想知道的是 GLM-4-Flash 的规模就是 9B 吗？效果和 GLM-3-Turbo 相比如何？

因为我们已经投入使用的模型就是 GLM-3-Turbo，GLM-4-Flash 的能力大于  GLM-3-Turbo 的话，我们就考虑替换了

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
不是glm-4-9b

**Reply by NianBroken** [reply]
那么他们之间有什么具体的区别吗？
